core/jazz_assets/config/global-config.json

```diff
@@ -81,7 +81,8 @@
 		"provider",
 		"provider_id",
 		"limit",
-		"offset"
+		"offset",
+		"status"
```

Comment:
Any swagger changes needed?

Reply:
No. Its already there in swagger,. Here only changing the field as searchable